### DISCUSSION TITLE
Make Loudness setting permanent?

### DISCUSSION THREAD

**Discussion by snuew** [Mr-MIBonk/M.I.B._More-Incredible-Bash#295] [Ideas]
Hi, I've installed an amp in the boot and new speakers in my 2016 Passat, and I have had an issue that makes songs sound really bad, and I've finally found the fix, and its the loudness setting and turning that off, and it sounds way better, but it re-enables after a reboot, do you have plans to make the Loudness setting permanent? so it says off even after reboot? i.e. when I turn the car off and on, Thanks!

![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/9547323/349a62f1-2e33-4193-88b7-1cb5994c2e5e)

**Comment by snuew** [comment]
Honesty I have been trying to figure out this issue with the audio where songs sound so compressed, and its been driving me insane, so hopefully you can make it so it can be disabled permanently sometime. I have the Helix Sub and I've swapped the door speaker out, sounds incredible with the loudness setting off, so I'm guessing its been compressing the audio by default, and sounds so bad to me.

**Comment by harman-f** [comment]
There is fixed solution. 
if you installed a custom sound system you have to adjust the sound datasets in the unit. 
they are now still applying the stock profiles made for the stock speakers. That will always sound bad 😉. 

There is a project on GIT dealing with MIB2 sound datasets.
